File: kaggle_environments/envs/kore_fleets/starter_bots/java/test/ConfigurationTest.java
package test;

import java.io.IOException;
import java.nio.file.Files;
import java.nio.file.Path;
import java.nio.file.Paths;

import org.junit.Assert;
import org.junit.Test;

import kore.Configuration;

public class ConfigurationTest {

    @Test
    public void givenValidConfiguration_createSuccessful() throws IOException {
        Path configPath = Paths.get("bin", "test", "configuration.json");
        String rawConfig = Files.readString(configPath);        
        
        Configuration config = new Configuration(rawConfig);

        Assert.assertEquals(0.02, config.regenRate, .001);
    }
    
}

File: kaggle_environments/envs/kore_fleets/starter_bots/ts/test/ConfigurationTest.ts
import { describe } from 'mocha';
import { expect } from 'chai';

import * as fs from 'fs';

import { Configuration } from "../kore/Configuration";

describe('Configuration', () =>  {
    it('init works correctly', () => {
        const rawConfig = fs.readFileSync('./test/configuration.json','utf8');
        
        const config = new Configuration(rawConfig);

        expect(config.regenRate).to.equal(.02);
    })
});